Packs/GitLab/Integrations/GitLabv2/GitLabv2.py

```diff
@@ -0,0 +1,1608 @@
+    outputs = partial_response([response], 'Merge Request') if return_partial else response
+    human_readable_str = 'Merge request was updated successfully.'
+    return CommandResults(
+        outputs_prefix='GitLab.MergeRequest',
+        outputs_key_field='iid',
+        readable_output=human_readable_str,
+        outputs=outputs,
+        raw_response=response
+    )
+
+
+def merge_request_note_list_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+    Returns a list of merge request's notes.
+    Args:
+        client (Client): Client to perform calls to GitLab services.
+        args (Dict[str, Any]): XSOAR arguments
+    Returns:
+        (CommandResults).
+    """
+    response_to_hr = []
+    headers = ['Id', 'Author', 'Text', 'CreatedAt', 'UpdatedAt']
+    page_number = arg_to_number(args.get('page')) or 1
+    limit = arg_to_number(args.get('limit')) or 50
+    merge_request_iid = args.get('merge_request_iid')
+
+    params = assign_params(sort=args.get('sort'), order_by=args.get('order_by'))
+    response = response_according_pagination(client.merge_request_note_list_request, limit, page_number, params,
+                                             merge_request_iid)
+
+    for merge_request_note in response:
+        merge_request_note_edit = {'Id': merge_request_note.get('id', ''),
+                                   'Text': merge_request_note.get('body', ''),
+                                   'UpdatedAt': merge_request_note.get('updated_at', ''),
+                                   'CreatedAt': merge_request_note.get('created_at', '')}
+        if merge_request_note.get('author'):
+            merge_request_note_edit['Author'] = merge_request_note.get('author', {}).get('name', '')
+        response_to_hr.append(merge_request_note_edit)
+    return_partial = args.get('partial_response') == 'true'
+    outputs = partial_response(response, 'Merge Request Note') if return_partial else response
+    human_readable = tableToMarkdown('List Merge Issue Notes', response_to_hr, removeNull=True, headers=headers)
+    return CommandResults(
+        outputs_prefix='GitLab.MergeRequestNote',
+        outputs_key_field='id',
+        readable_output=human_readable,
+        outputs=outputs,
+        raw_response=response
+    )
+
+
+def merge_request_note_create_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+    Creates a merge request note.
+    Args:
+        client (Client): Client to perform calls to GitLab services.
+        args (Dict[str, Any]): XSOAR arguments:
+            - 'merge_request_iid: the IID of the merge request.
+            - 'body':  Create notes with a description.
+    Returns:
+        (CommandResults).
+    """
+    merge_request_iid = args.get('merge_request_iid')
+    body = args.get('body')
+    response = client.merge_request_note_create_request(merge_request_iid, body)
+    human_readable_str = 'Merge request note created successfully.'
```

Comment:
Let's write the string directly in the readable_outputs in the CommandResults 
(Only if it's just string) - in all relevant places

Reply:
done